extensions/standard-processors/processors/GetFile.cpp

```diff
@@ -244,7 +244,7 @@ bool GetFile::fileMatchesRequestCriteria(std::string fullName, std::string name,
     return false;
 
   utils::Regex rgx(request.fileFilter);
-  if (!utils::regexSearch(name, rgx)) {
```

Comment:
This is a breaking change, as shown by the unit test.  I think changing the default value of `FileFilter` to `.*` (which it really [almost] was, with `regexSearch`) would lessen its impact.

Reply:
You are right it's a breaking change, but the reason I changed it is to be aligned with the NiFi's implementation, to have the same default value and use the same regex matching.

Reply:
I don't know how closely we want to stay with the NiFi implementation, but I agree it would be better to have `.*` as the default. I'm okay with both, I wonder what the others think about this.

Reply:
I'm fine any of these: `.*`, `[^\\.].*`, `[^.].*`